From 6b648396e700db8e9753725e092a41040d8b9a5c Mon Sep 17 00:00:00 2001
From: "Rick Farina (Zero_Chaos)" <zerochaos@gentoo.org>
Date: Mon, 31 Jul 2023 23:33:36 -0400
Subject: [PATCH] fix -Wstringop-overread error

We are reading this 26 bytes into a 32 byte buffer, is this even any
worse?
Fixes: https://github.com/RfidResearchGroup/proxmark3/issues/2060
---
 client/src/mifare/mifarehost.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client/src/mifare/mifarehost.c b/client/src/mifare/mifarehost.c
index a05456bcd6..76c392490d 100644
--- a/client/src/mifare/mifarehost.c
+++ b/client/src/mifare/mifarehost.c
@@ -787,7 +787,7 @@ int mfStaticNested(uint8_t blockNo, uint8_t keyType, uint8_t *key, uint8_t trgBl
         p_keyblock = mem;
     }
 
-    uint8_t fn[26] = "static_nested_000.bin";
+    uint8_t fn[32] = "static_nested_000.bin";
 
     uint64_t start_time = msclock();
     for (uint32_t i = 0; i < keycnt; i += max_keys_chunk) {
